cmd/nvidia-dra-plugin/nvlib.go

```diff
@@ -495,11 +511,19 @@ func (l deviceLib) setTimeSlice(uuids []string, timeSlice int) error {
 }
 
 func (l deviceLib) setComputeMode(uuids []string, mode string) error {
+	// In order for nvidia-smi to run, we need to set the PATH to the parent of
```

Comment:
Does it make sense to add a `runNvidiaSMI` method to encapsulate this logic?

https://github.com/NVIDIA/k8s-dra-driver/pull/111/files#diff-37cd66635c91c8a11b5779dcb053d97a88f6ea32c42d1b1645a998cb03b1b8deR506-R517

Reply:
i thought about it, but purposefully didn't since there are only 2 instances of this call  and we want to drop the second one soon.